rgw/v2/tests/s3_swift/configs/test_sse_s3_per_object_with_sts.yaml

```diff
@@ -0,0 +1,36 @@
+# polarion test case id: CEPH-83586489
+# test scripts : test_sts_using_boto.py
+config:
```

Comment:
Please update this config in script file mentioned here

Reply:
done @anrao19